src/param.c

```diff
@@ -43,17 +43,19 @@ void set_param_defaults(void)
     sprintf(temp_name, "TEMP_%c%c", 'A' + id/10, 'A' + id%10);
     init_param(id, temp_name, id);
   }
-  init_param(PARAM_BOARD_REVISION, "BOARD_REV", 5);
+  init_param(PARAM_BOARD_REVISION, "BOARD_REV", 4);
+
+  init_param(PARAM_BAUD_RATE, "BAUD_RATE", 921600);
 
   init_param(PARAM_SYSTEM_ID, "SYS_ID", 1);
   init_param(PARAM_STREAM_HEARTBEAT_RATE, "STRM_HRTBT", 1);
-  init_param(PARAM_STREAM_IMU_RATE, "STRM_IMU", 100);
+  init_param(PARAM_STREAM_IMU_RATE, "STRM_IMU", 500);
```

Comment:
Do we want the default that high? I'd think we should keep it at 100 since that's sufficient for most applications and 500 might clog up the line?